packages/bundler/tsconfig.json

```diff
@@ -1,4 +1,7 @@
 {
   "extends": "@frontside/tsconfig",
-  "files": ["src/index.ts"]
+  "files": ["src/index.ts"],
+  "compilerOptions": {
+    "esModuleInterop": true
+  }
```

Comment:
I'm don't think this tsconfig.json will get picked up when running tests in a downstream project using BigTest. This likely works because this is the tsconfig.json that gets used when running the Bundler tests. We may need to specify this option directly in the `@babel/plugin-transform-typescript` config.

Reply:
This is actually so that the import of `injectProcessEnv` works from the process.env rollup plugin and can be used in our pipeline, not for the typescript in the pipeline itself if that's what you mean. basically, I couldn't get the `bundler.ts` to compile without this since `import * as plugin from 'rollup-plugin-inject-process-env'`  wouldn't work.

Reply:
In summary, I need to change the tsconfig.json so that `bundler.ts` compiles, not so that the test bundle compiles.

Reply:
Ah, that's right. Okay, this should be fine then.